### PR TITLE
Fixing sqldb tests

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by andrewconner** [comment]
@eishay @leogrim

File: kifi-backend/modules/shoebox/app/views/marketing/terms.scala.html [hunk 1]
```diff
@@ -259,7 +259,7 @@ <h2 id="claims-of-copyright-or-trademark-infringement">Claims of Copyright or Tr
 FortyTwo Inc.<br>
 883 N Shoreline Blvd Suite A200<br>
 Mountain View, CA 94043<br>
-Tel: 650-605-8112<br>
+Tel: 650-963-9967<br>
```
Comment (eishay) [inline kifi-backend/modules/shoebox/app/views/marketing/terms.scala.html:262]: :-)

File: kifi-backend/modules/sqldb/test/com/keepit/common/db/DataBaseComponentTest.scala [hunk 2]
```diff
@@ -103,18 +103,6 @@ class DataBaseComponentTest extends Specification with DbTestInjector {
       }
     }
```
Comment (eishay) [inline kifi-backend/modules/sqldb/test/com/keepit/common/db/DataBaseComponentTest.scala:105]: why did you remove this on?

Comment (andrewconner) [inline kifi-backend/modules/sqldb/test/com/keepit/common/db/DataBaseComponentTest.scala:105]: @leogrim said it's not needed, because that functionality doesn't exist anymore.

Comment (eishay) [inline kifi-backend/modules/sqldb/test/com/keepit/common/db/DataBaseComponentTest.scala:105]: ok

Comment (leogrim) [inline kifi-backend/modules/sqldb/test/com/keepit/common/db/DataBaseComponentTest.scala:105]: We removed not retrying on IntegrityConstraint exceptions after we set up the mechanism to clear the cache on such occasions.
